lib/kdump_utils.pm

```diff
@@ -27,10 +27,12 @@ our @EXPORT = qw(install_kernel_debuginfo prepare_for_kdump
 
 sub install_kernel_debuginfo {
     zypper_call 'ref';
-    # JeOS uses kernel-default-base, except on aarch64 openSUSE
-    my $kernel    = script_output('rpm -qf --qf %{name} /boot/initrd-$(uname -r)');
-    my $debuginfo = script_output('rpmquery --queryformat="%{NAME}-%{VERSION}-%{RELEASE}\n" ' . $kernel . '| sort --version-sort | tail -n 1');
+    my $kernel = script_output('rpm -qf --qf %{name} /boot/initrd-$(uname -r)');
+    my $debuginfo;
+    $debuginfo = script_output('rpmquery --queryformat="%{NAME}-%{VERSION}-%{RELEASE}\n" ' . $kernel . '| sort --version-sort | tail -n 1');
+    # on jeos of some products there is no 'kernel-default-base-debuginfo', use 'kernel-default-debuginfo' instead.
     $debuginfo =~ s/$kernel/$kernel-debuginfo/g;
+    $debuginfo = 'kernel-default-debuginfo' if ((is_sle('>=15-sp2') || is_tumbleweed || is_leap('>=15.2')) && is_jeos && ($kernel eq 'kernel-default-base'));
```

Comment:
This will overwrite the version.

Could you try this?:

```suggestion
    if ((is_sle('>=15-sp2') || is_tumbleweed || is_leap('>=15.2')) && is_jeos && ($kernel eq 'kernel-default-base')) {
        $debuginfo =~ s/-base//';
    }
```

Since this condition is very long, I would prefer a standard if statement or a function like `is_minimal_kernel` or somethink like that

`$debuginfo =~ s/-base//' if !is_minimal_kernel();`